go/coinstacks/thorchain/api/handler.go

```diff
@@ -50,6 +53,97 @@ func (h *Handler) GetAccount(pubkey string) (api.Account, error) {
 	return a, nil
 }
 
+func (h *Handler) GetTxHistory(pubkey string, cursor string, pageSize int) (api.TxHistory, error) {
+	// attempt to find matching fee event for txid or use default fee as defined by https://daemon.thorchain.shapeshift.com/lcd/thorchain/constants
+	matchFee := func(txid string, events []thorchain.TypedEvent) cosmos.Value {
+		for _, e := range events {
+			switch v := e.(type) {
+			case *thorchain.EventFee:
+				if txid == v.TxID {
+					return thorchain.CoinToValue(v.Coins)
+				}
+			}
+		}
+
+		return cosmos.Value{Amount: "2000000", Denom: "rune"}
```

Comment:
check out `NativeTransactionFee` from `/thorchain/constants` on the lcd. it's not subject to change often so def not a blocker